packages/frontend/src/providers/DashboardProvider.tsx

```diff
@@ -115,6 +118,8 @@ export const DashboardProvider: React.FC = ({ children }) => {
 
     const [invalidateCache, setInvalidateCache] = useState<boolean>(false);
 
+    const [chartSort, setChartSort] = useState<Record<string, SortField[]>>({});
```

Comment:
Why do we store the state on the dashboard provider instead of something at the tile level like `useDashboardChart`? @IrakliJani or @joaoviana could confirm, but if it's here it could trigger re-render of all charts when sorting changes (since all charts depend on the chart sort object).

Reply:
once this is rebased, I can have a look on Render environment and check with the Performance profiler, sounds good @owlas  👍

Reply:
we should also be able to `share` this later https://github.com/lightdash/lightdash/issues/7948 , in case this is a blocker for moving the logic out of the dashboardProvider

Reply:
This logic seems to be affecting other charts in the dashboard, at least they are not re-rendered. But let's confirm this  
[Screencast from 15-11-23 12:13:10.webm](https://github.com/lightdash/lightdash/assets/1983672/e026eb06-a6f5-47f2-ae21-828884bb3ccf)

Reply:
I can confirm that the paint flashes only target the table you're sorting, so it looks good! 💪 

https://github.com/lightdash/lightdash/assets/7611706/2c3d99b4-7c60-4ca3-a863-47732745d892